gammapy/scripts/catalog_query.py

```diff
@@ -2,6 +2,7 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 import logging
 import click
+click.disable_unicode_literals_warning = True
```

Comment:
Concerning
```
click.disable_unicode_literals_warning = True
```

Please add a short entry "Click" to the Gammapy developer howto:
https://gammapy.readthedocs.org/en/latest/development/howto.html
For now it can be just one sentence saying that this line should be added to any command line tool that uses click and a link to  http://click.pocoo.org/5/python3/#unicode-literals that explains what it is.

You could also mention the warnings one gets if this is not added, so that if someone sees the warning, they find that entry via the full-text search in the docs.

It would be nice if you apply the same line for the few other tools where we use click at the moment:
https://github.com/gammapy/gammapy/search?utf8=%E2%9C%93&q=import+click
Splitting this out in a separate PR would be nice, but do as you like.